### PR TITLE
[WIP] Owners module

### PR DESCRIPTION
**Pull request by relekang**
This will introduce a model that can hold default settings for projects. A common use case, an organisation wants to use a specific queue, but do not want to set it for all projects manually.

### CODE REVIEW

File: frigg/helpers/github.py [hunk 1]
```diff
@@ -143,7 +150,9 @@ def api_request(url, token, data=None, page=None):
         }
         response = requests.post(url, data=json.dumps(data), headers=headers)
 
+    print(url)
+
     if settings.DEBUG:
         print((response.headers.get('X-RateLimit-Remaining')))
-
+    print(response.text)
```
Comment (frecar) [inline frigg/helpers/github.py]: remove print? :)

File: frigg/owners/migrations/0002_add_from_projects.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from django.db import migrations
+
+
+def create_owners_from_projects(apps, schema_editor):
+    Project = apps.get_model("builds", "Project")
+    Owner = apps.get_model("owners", "Owner")
+    for project in Project.objects.all():
+        Owner.objects.create(
```
Comment (frecar) [inline frigg/owners/migrations/0002_add_from_projects.py]: is this autogenerated? get_or_create instead?

File: frigg/helpers/github.py [hunk 3]
```diff
@@ -21,6 +21,13 @@ def get_commit_url(build):
     )
 
 
+def list_members(owner):
+    url = 'orgs/{}/members'.format(owner.name)
+    data = json.loads(api_request(url, owner.github_token).text)
+    print(data)
```
Comment (frecar) [inline frigg/helpers/github.py]: print

File: frigg/helpers/github.py [hunk 4]
```diff
@@ -143,7 +150,9 @@ def api_request(url, token, data=None, page=None):
         }
         response = requests.post(url, data=json.dumps(data), headers=headers)
 
+    print(url)
```
Comment (frecar) [inline frigg/helpers/github.py]: print

**Comment by frecar** [comment]
Sweet :) Should we add a test for this as well?

**Comment by relekang** [comment]
> Sweet :) Should we add a test for this as well?

I think we should add several tests, or was this intended at an inline comment to something specific?

**Comment by frecar** [comment]
I agree that we should add several tests, it was more like an attempt to ask politely :)

**Comment by relekang** [comment]
:+1:

**Pull request closed without merging**